src/settings.cpp

```diff
@@ -20,11 +20,34 @@
 #include "settings.h"
 
 #include <QtCore/QSettings>
-#include <QDebug>
+#include <QStandardPaths>
+#include <QCoreApplication>
+#include <QFile>
+#include <QDir>
 
 Settings::Settings(QObject *parent) :
-    QObject(parent), m_settings(new QSettings(this))
-{
+    QObject(parent)
+{
+    const QString settings_path =
+        QStandardPaths::writableLocation(QStandardPaths::AppConfigLocation)
+        + "/" + QCoreApplication::applicationName() + ".conf";
+    QFile settings_file(settings_path);
+
+    // Migrate old settings and delete them
+    const QString old_settings_path =
+            QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) +
+            "/harbour-quickddit/harbour-quickddit.conf";
```

Comment:
Will this be .config/$org/$app/harbour-quickddit/harbour-quickddit.conf in the end?

The author of this forked changed the appname from harbour-quickddit to Quiddit.

Reply:
No, this evaluates to just ".config/harbour-quickddit/harbour-quickddit.conf". The path you described would be with `QStandardPaths::AppConfigLocation`.